public/components/content-list/content-list.html

```diff
@@ -17,7 +17,7 @@
         </tr>
     </thead>
 
-    <tbody class="content-list__item-container" ng-repeat="group in content track by group.name" content-list-item-container>
+    <tbody data-cy="content-list-{{group.name}}" class="content-list__item-container" ng-repeat="group in content track by group.name" content-list-item-container>
```

Comment:
Resolves to this: ![image](https://user-images.githubusercontent.com/25747336/91306196-1fa85600-e7a4-11ea-9ade-f049828cec0a.png)